docs/reference/upload-rights.rst

```diff
@@ -0,0 +1,120 @@
+Upload Rights
+#############
+
+Those who are allowed to upload the kernel to the Ubuntu archive take their
+responsibility very seriously.
+
+To obtain per package upload rights for the Ubuntu kernel, you can apply to
+become a member of the ubuntu-kernel-uploaders_ team in Launchpad.
+
+People can join this team only after going through a thorough application and
+review process. The sections below describe the general member profile of the
+ubuntu-kernel-uploaders_ team, as well as the application process.
+
+Member Profile
+==============
+
+Below is the general profile for those having per package upload rights for the
+Ubuntu kernel.
+
+- Members of the ubuntu-kernel-uploaders_ Launchpad team
+
+- Generally have commit access to the `Ubuntu kernel git repositories`_
+
+- Actively follow and participate in discussions and patch reviews on the
+  `Ubuntu kernel-team mailing list`_
+
+- Are collectively responsible for the maintenance of packages in the Ubuntu
+  kernel package set for all supported releases as well as the development
+  release
+
+- Have a strong working knowledge of kernel packaging concepts and techniques,
+  refined through experience
+
+- Have a strong working knowledge of Ubuntu project procedures, especially
+  those related to the release process and support commitments, and an
+  understanding of the reasons why they exist
+
+- Have a history of substantial direct contributions to the distribution and
+  particularly the kernel related packages
+
+- Feel a sense of personal responsibility for the quality of Ubuntu releases
+  and for the satisfaction of Ubuntu users
+
+- Exercise great care in their work, with the understanding that their efforts
+  have a direct impact on others, including:
+
+  - every Ubuntu user
+  - the Ubuntu release team
+  - corporate partners who provide support for Ubuntu
+
+Application Process
+===================
+
+As alluded to in the member profile above, membership consideration for the
+ubuntu-kernel-uploaders team adheres to a strict policy. Anyone considering
```

Comment:
i'm not sure if you want to link to the launchpad  team here since the previous 2 references to 'ubuntu-kernel-uploaders' team is linked.